pkg/manager/impl/execution_manager.go

```diff
@@ -444,8 +444,11 @@ func (m *ExecutionManager) getClusterAssignment(ctx context.Context, request *ad
 	if resource != nil && resource.Attributes.GetClusterAssignment() != nil {
 		return resource.Attributes.GetClusterAssignment(), nil
 	}
-	// Defaults to empty assignment with no selectors
-	return &admin.ClusterAssignment{}, nil
+	clusterPoolAssignment := m.config.ClusterPoolAssignmentConfiguration().GetClusterPoolAssignments()[request.GetDomain()]
+
+	return &admin.ClusterAssignment{
+		ClusterPoolName: clusterPoolAssignment.Pool,
```

Comment:
What happens in case this comes as an empty pool name

Reply:
Existing behavior is preserved, we just don't assign a clusterpool which is fine.